problems/kattis/A-New-Alphabet/java/sayloradm/TestFinal.java

```diff
@@ -0,0 +1,22 @@
+package secondPackage;
+
+import firstPackage.newAlphabet;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class TestFinal {
+
+	@Test
+	public void test() {
+		
+		newAlphabet tester = new newAlphabet();
+		
+		String out = "\\/\\/0|Z|)$";
+		
+		assertEquals(out,tester.translate("words"));
+	}
+	
+
+}
```

Comment:
nit - extra white space lines.  Refer to java style guidelines and do some cleanup.

Reply:
Removed some white space!